### ISSUE TITLE
FollowPath: execute path only once

### ISSUE THREAD

**Issue by janpascal** [PokemonGoF/PokemonGo-Bot#4663]
### Short Description
Sometime I prefer the bot to follow the path only once, instead of looping. This could be useful for having the bot walk to a new location following a number of waypoints, and then go chasing pokemon and pokestops at the new location.

### Possible solution
In the FollowPath configuration, implement path_mode = 'single'. In follow_path.py in the work() function, check for this path_mode in the piece of code that handles the situation where the end of the path is reached. If so, return WorkerResult.SUCCESS to allow lower priority tasks to run.

### How it would help others
It would help if you want your bot to walk to another location once, and there in the new area after the path is finished.

**Comment by MZorzy** [comment]
Or add something like 'stop' on map.json
(and pause tot sec)

**Comment by maxprzemo** [comment]
A similar effect can be achieved now.
Set the number of laps
          "number_lap": 1,
A restart time eg
          "timer_restart_min": "06:00:00",
          "timer_restart_max": "07:00:00"
After each lap be a break 6-7 hours

**Comment by janpascal** [comment]
Not exactly : in that case the bot would stop completely instead of going off chasing pokestops (if you put in a Move to Tower with a lower priority) 

On 24 August 2016 18:08:45 CEST, "Przemysław " <notifications@github.com> wrote:
>A similar effect can be achieved now.
>Set the number of laps
>          "number_lap": 1,
>A restart time eg
>          "timer_restart_min": "06:00:00",
>          "timer_restart_max": "07:00:00"
>After each lap be a break 6-7 hours
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/PokemonGoF/PokemonGo-Bot/issues/4663#issuecomment-242121099

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
